progress.go

```diff
@@ -11,27 +13,48 @@ const (
 )
 
 type TableProgress struct {
+	TableName                   string
+	PaginationKeyName           string
 	LastSuccessfulPaginationKey uint64
 	TargetPaginationKey         uint64
-	CurrentAction               string // Possible values are defined via the constants TableAction*
+	Status                      string // Possible values are defined via the constants TableAction*
 	RowsWritten                 uint64
 }
 
 type Progress struct {
+	GhostferryVersion string
+
+	SourceHostPort string
+	TargetHostPort string
+
+	StartTime   time.Time
+	CurrentTime time.Time
+	TimeTaken   time.Duration
+
+	AllTableNames    []string
```

Comment:
Adding these will significantly balloon the progress dump if you have a lot of tables, which may cause issues with downstream consumers. These table names never change during a run so there is no reason for it to be attached to the `Progress` object.